modules/ROOT/pages/fd-share-to-studio.adoc

```diff
@@ -6,28 +6,34 @@ endif::[]
 
 :imagesdir: ../assets/images
 
-When you are creating a Mule application, there might come a time when you would like help from someone in your Anypoint-Platform organization, such as a developer, who works in Studio. You can download your project as a `.jar` file and then place that file in a location from which the other person can import it into Studio.
+Perhaps when you are creating a Mule application you need help from someone in your Anypoint-Platform organization,
+such as a developer, who works in Studio. You can download your project as a `.jar` file and then place that file in a location from which the other person can
+import it into Studio.
 
-As the other person works on the project, the changes are applied to your project online in Flow Designer. When the other person is finished, you can open your project in Flow Designer and pick up where that person left off.
+You'll need to use *Export project sources* to share just your project source files (including all the connector credentials) without all of the other artifacts that are required by the Mule runtime.
 
-== Procedure
+As the other person works on the project, the changes are applied to your project online in Flow Designer.
```

Comment:
@mcherem this line is confusing to me. there is no link between what's happening offline in studio and the flow designer project online. Also because Studio offers more capabilities today - it isn't always true that the user can then re-import the app back into FD after some changes have been made in Studio land

Reply:
Fixed.